ask_chatgpt.gemspec

```
@@ -13,9 +13,11 @@ Gem::Specification.new do |spec|
   spec.metadata["homepage_uri"] = spec.homepage
 
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
+    Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md", "bin/ask_chatgpt"]
   end
 
+  s.executables   = ["ask_chatgpt"]
```

This probably should be `spec`
```suggestion
  spec.executables   = ["ask_chatgpt"]
```